flit/__init__.py

```diff
@@ -22,6 +23,7 @@ def find_python_executable(python):
         return sys.executable
     if os.path.isabs(python):  # sys.executable is absolute too
         return python
+    python = shutil.which(python) or os.path.abspath(python)
```

Comment:
Why the `abspath` call? I think `which()` already handles relative paths.

Reply:
I was not aware of this, thanks! TIL 😄

Reply:
What should we fall back to if the `which` call fails though? (It returns `None` if there’s no matching executable found.) Just the unresolved `python`?

Reply:
We should probably throw an error, I'd guess - that means you've given it a name which isn't on PATH, or a path which doesn't exist or isn't executable.

Reply:
I added an exception for this.